src/corehost/cli/json.h

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __JSON_H__
+#define __JSON_H__
+
+#include <memory>
+#include "pal.h"
+#include "rapidjson/document.h"
+#include "rapidjson/fwd.h"
```

Comment:
Maybe I'm missing it, but do the types defined in `fwd.h` always use `char` as the character type instead of the whole platform-dependent `char`/`wchar` thing we do? (I'm not familiar with RapidJSON at all - just searched for one of the types being used in this change)

Reply:
Yes, you're right.  I've declared the types we need inside `json_parser_t` to take into account the different encodings and character type width.